### DISCUSSION TITLE
Welcome to Bleak Discussions!

### DISCUSSION THREAD

**Discussion by hbldh** [hbldh/bleak#439] [General]
## 👋 Welcome to the Bleak discussion space!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

A lot of issues that gets reported are not errors in Bleak per se, but fall in one or more of the following categories:

- Problems with connecting to a device.
- Problems with how to ensure a clean disconnect
- Problems with how to enable reconnections
- Problems with how different OS backends behaves and handles devices.
- Questions about whether a specific functionality is present or will be present in the future

These kinds of topics is better kept here, leaving the issues for actual bugs and errors. If your issue gets moved here, do not regard it as the Bleak maintainers not dealing with it, but rather trying to differentiate between things that need to be fixed and things that need to be answered and discussed.

Here are some resources that might help you, before you ask questions here:

Tutorial on asyncio: https://realpython.com/async-io-python/
Tutorial on BLE and GATT: https://learn.adafruit.com/introduction-to-bluetooth-low-energy/introduction
Examples for Bleak: https://github.com/hbldh/bleak/tree/develop/examples
Documentation for Bleak: https://bleak.readthedocs.io/en/latest/

**Comment by hbldh** [comment]
@dlech I aim to try to use Discussions for things that gets posted as issues but are more of a question or cry for help. Feel free to migrate such issues here so we can get an overview of what is actually issues to solve and what is not. How does that sound?

**Reply by dlech** [reply]
Sounds good to me.
